payment-gateway/go-example/main.go

```diff
@@ -151,8 +152,8 @@ func userDoGaslessPayment(receiver common.Address, amount *big.Int) {
 		log.Fatalf("Failed to create ERC20 transfer data: %v", err)
 	}
 
-	// Get the latest nonce for the from address
-	nonce, err := client.PendingNonceAt(context.Background(), fromAddress)
+	blockNumber := rpc.PendingBlockNumber
+	nonce, err := paymasterClient.GetTransactionCount(context.Background(), fromAddress, rpc.BlockNumberOrHash{BlockNumber: &blockNumber})
```

Comment:
as you change from `client` to `paymasterClient`, the `client` is no longer needed , the config as well, please delete it